tmt/cli/__init__.py

```diff
@@ -139,7 +139,7 @@ def pass_context(fn: 'Callable[Concatenate[Context, P], R]') -> 'Callable[P, R]'
     return click.pass_context(fn)  # type: ignore[arg-type]
 
 
-@dataclasses.dataclass
+@dataclasses.dataclass  # noqa: TID251
```

Comment:
This could be `@container`.

Reply:
The reason why I didn't convert those is to make it easier to extend the container types, e.g. for parsing docstrings. The cases where I didn't convert is because they do not seem to have a link to the `.fmf` entries.